pkg/types/rfc3161/v0.0.1/entry.go

```diff
@@ -210,3 +210,15 @@ func (v V001Entry) CreateFromArtifactProperties(_ context.Context, props types.A
 func (v V001Entry) Verifier() (pki.PublicKey, error) {
 	return nil, errors.New("Verifier() does not support rfc3161 entry type")
 }
+
+func (v V001Entry) Insertable() (bool, error) {
```

Comment:
Should we specify writeOnly for TSR? https://github.com/sigstore/rekor/blob/main/pkg/types/rfc3161/v0.0.1/rfc3161_v0_0_1_schema.json#L15

More generally, do we want to be consistent with adding writeOnly tags to all types?

Reply:
It’s not writeOnly because it will be read out as well